allauth/account/views.py

```diff
@@ -145,6 +147,8 @@ class LoginView(
     redirect_field_name = "next"
 
     @sensitive_post_parameters_m
+    @method_decorator(csrf_protect)
```

Comment:
**Issue**: Why is `csrf_protect` explicitly added here? It would assume you're running without the CSRF middleware turned on, but then this decorator is not sufficient, there are other forms as well that would need it. Also, adding this goes beyond scope of this PR -- can you remove this line?